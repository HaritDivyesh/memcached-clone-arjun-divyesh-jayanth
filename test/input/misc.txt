1
flush_all
1
get key
2
set key 1 0 2
hi
2
set mykey 1 0 3
hey
1
flush_all
1
get key mykey
1
version
1
quit
