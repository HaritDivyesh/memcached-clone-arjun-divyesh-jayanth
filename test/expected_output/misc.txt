flush_all
OK
get key
END
set key 1 0 2
hi
STORED
set mykey 1 0 3
hey
STORED
flush_all
OK
get key mykey
END
version
VERSION 0.0.1
quit
